server/apiCalls/transformers/memosCount.test.js

```diff
@@ -9,6 +9,7 @@ const offering1WithDates = ({ dates = {}, courseCode = 'SF1625', memoValues = {}
     courseCode,
     isPdf: false,
     ladokRoundIds: ['1'],
```

Comment:
`ladokRoundIds`  has to be removed to be sure tests working properly

Reply:
Removed